eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/SlugConverter.php

```diff
@@ -74,7 +73,6 @@ class SlugConverter
                     'endline_search_normalize',
                     'tab_search_normalize',
                     'specialwords_search_normalize',
-                    'punctuation_normalize',
```

Comment:
Right. :) I'm not sure that is correct. I believe the "urlalias_compat" did earlier filter out commas, and that it still should do so, using the "punctuation_normalize". I haven't tested this though.